src/oic/oic/provider.py

```diff
@@ -258,15 +262,15 @@ def __init__(
             keyjar=keyjar, verify_ssl=verify_ssl, message_factory=message_factory
         )
         # Same keyjar
-        self.keyjar = self.server.keyjar
+        self.keyjar = self.server.keyjar  # type: KeyJar
 
         self.endp.extend([UserinfoEndpoint, RegistrationEndpoint, EndSessionEndpoint])
 
         self.userinfo = userinfo
         self.template_renderer = template_renderer
         self.baseurl = baseurl or name
-        self.cert = []
-        self.cert_encryption = []
+        self.cert = []  # type: ignore  # UNUSED?
```

Comment:
Probably unused, doesn't ring a bell here either. If those are not used in the examples, they could go.

Reply:
OK, it turns out it is used in example1:
https://github.com/OpenIDC/pyoidc/blob/699cc99850d26565d3fbdc6357ca85a56bd6e5e9/oidc_example/op1/claims_provider.py#L186-L188

But it looks like it shouldn't be used in this way and rather configured statically? It is not set anywhere in the example...

Reply:
Yes, that should just be some static resource.